config/release-41x-211.conf

```diff
@@ -0,0 +1,95 @@
+##### Defined in the script #####
+# CURRENT_DIR: execution dir
+#####
+
+##### local dirs #####
+# Unlikely to need modification
+#ZINC_DIR=${CURRENT_DIR}/zinc
+#SCALATEST_DIR=${SCRIPT_DIR}/scalatest
+#BUILD_DIR=${CURRENT_DIR}/target
+#LOCAL_M2_REPO=${BUILD_DIR}/m2repo
+#P2_CACHE_DIR=${BUILD_DIR}/p2cache
+#KEYSTORE_DIR=${BUILD_DIR}/keystore
+#SCALA_DIR=${BUILD_DIR}/scala
+#ZINC_BUILD_DIR=${BUILD_DIR}/zinc
+#SCALA_IDE_DIR=${BUILD_DIR}/scala-ide
+#SCALA_REFACTORING_DIR=${BUILD_DIR}/scala-refactoring
+#SCALARIFORM_DIR=${BUILD_DIR}/scalariform
+#WORKSHEET_PLUGIN_DIR=${BUILD_DIR}/worksheet
+#PLAY_PLUGIN_DIR=${BUILD_DIR}/play
+#SEARCH_PLUGIN_DIR=${BUILD_DIR}/search
+#PRODUCT_DIR=${BUILD_DIR}/product
+#SCALATEST_BUILD_DIR=${BUILD_DIR}/scalatest
+#PRODUCT_DIR=${BUILD_DIR}/product
+
+##### what to do #####
+# operation to perform (release, release-dryrun, nightly, scala-pr-validator, scala-pr-rebuild)
+OPERATION=release-dryrun
+# logging type (console, file)
+LOGGING=console
+# with caching? (true or false)
+WITH_CACHE=true
+# plugins to build: "worksheet play search" (between double quotes, space separated)
+PLUGINS="worksheet play search"
+# build the product (true or false)
+PRODUCT=true
+# Eclipse version (indigo, juno)
+ECLIPSE_PLATFORM=luna
+# type of build (dev, stable)
+BUILD_TYPE=stable
+
+##### Scala information #####
+SCALA_GIT_REPO=git://github.com/scala/scala
+# The version of Scala to use
+SCALA_VERSION=2.11.6
+# The version of Scala to use for multi-version support
+SCALA210_VERSION=2.10.5
+# The git version of Scala to use
+#SCALA_GIT_HASH=
+
+##### zinc/sbt information #####
+SBT_VERSION=0.13.6
+SBT_TAG=v0.13.6
```

Comment:
Someone said that 0.13.8 is not "stable" for Scala IDE, but I can't remember where it was said. @dragos Do you know more?